tests/automation/types/testing.ts

```diff
@@ -21,14 +21,14 @@ export type DMTimeOption =
   | 'time-option-10-seconds'
   | 'time-option-30-seconds'
   | 'time-option-60-seconds'
-  | 'time-option-300-seconds'
-  | 'time-option-1800-seconds'
-  | 'time-option-3600-seconds'
-  | 'time-option-21600-seconds'
-  | 'time-option-43200-seconds'
-  | 'time-option-86400-seconds'
-  | 'time-option-604800-seconds'
-  | 'time-option-1209600-seconds';
+  | 'time-option-5-minutes'
+  | 'time-option-30-minutes'
+  | 'time-option-1-hours'
```

Comment:
should just be 1 hour ... and same as 1 day (not hours or days)

Reply:
but its not important

Reply:
I really don't think we care. Those are not user facing, not read by screen readers on desktop